Homework_RalucaHotea/Challenge4/app/src/main/java/com/example/homework4/MainActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.example.homework4;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.util.Log;
+import android.os.Bundle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity {
+
+    private RecyclerView recyclerViewStudents;
+    private List<Student> students;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.my_recycler_view);
+        displayStudentList();
+    }
+
+    private void inbox() {
```

Comment:
The name of this method is not relevant to this context. Have you considered finding a more relevant one for students sample?